docs/README-docs.md

```diff
@@ -74,7 +74,7 @@ To add new pages to the book:
 | `src/chapter-name/...` | Top-level folders with high-level information: _architecture_, _technologies_, _build configurations_, etc.|
 | `src/chapter-name/section-name/...`, `src/chapter-name/subsection-name/...` | Documentation related to the special topic. Use subsections within the section when the subject changes, but you are still writing about a particular aspect of a larger subject. Note that both section and subsection files are in the chapter folder. |
 
-1. Put images into the `src/img` folder. We make diagrams with [draw.net](https://www.diagrams.net/).
+1. Put images into the `src/img` folder. We make diagrams with [diagrams.net](https://www.diagrams.net/).
```

Comment:
Here we may want to mention that `draw.io` can be also used offline as local app.

Reply:
I will add it with the next iteration.